doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.postprocess.TransformSourceTableTask.rst

```diff
@@ -0,0 +1,168 @@
+.. lsst-task-topic:: lsst.pipe.tasks.postprocess.TransformSourceTableTask
+
+########################
+TransformSourceTableTask
+########################
+
+
+``TransformSourceTableTask`` transforms the full source catalog
```

Comment:
So there is some potential terminology confusion with "source catalog" (which I personally would read to be an afwTable source catalog, as it currently gets output by processCcd) and a "source table" which to me could mean two things, one being a parquet table derived from the source catalog (`source`) and the other being a Source Table (™️) (`sourceTable`).  Maybe phrase this as "transforms the full table of sources (a "source" dataset) to a DPDD..."